recipe/meta.yaml

```diff
@@ -1,61 +1,55 @@
 {% set name = "certifi" %}
-{% set version = "2025.7.14" %}
-
-{% set pip_version = "24.3.1" %}
-{% set setuptools_version = "75.6.0" %}
+{% set version = "2025.8.3" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://github.com/certifi/python-certifi/archive/refs/tags/2025.07.14.tar.gz
-    sha256: 89734fb727880c3ddc37ad526b6de23159188c41468d6d5a0fab92642688bd46
-    folder: {{ name }}
-  # bootstrap pip and setuptools to avoid circular dependency
```

Comment:
Please keep the previous build files. These are needed when building for a new python version.